realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -150,12 +150,14 @@
 
     @Override
     protected void finalize() throws Throwable {
+        /*
```

Comment:
Yes! But it seems this should be in the `BaseRealm` since currently we missed the log in `DynamicRealm`'s finalizer. Will fix it later.